src/Files.App/Helpers/Win32/Win32PInvoke.Methods.cs

```diff
@@ -502,5 +502,22 @@ int dwFlags
 			string pszRootPath,
 			ref SHQUERYRBINFO pSHQueryRBInfo
 		);
+
+		[DllImport("shell32.dll")]
+		static extern int SHGetKnownFolderPath(
```

Comment:
Please use CsWin32…
If we in rush, no problem, I’ll do that instead.

Reply:
Sounds good.